docs/mining/proof-of-work.md

```diff
@@ -12,9 +12,9 @@ are made. These coins are then split up as follows:
 
 Subsidy | Party
 ---     | ---
-`60%`   | PoW Miners
-`30%`   | PoS Voters
-`10%`   | Decred development subsidy
+60%   | PoW Miners
```

Comment:
Nope just a small improvement in appearance and in making our use of markdown more correct/consistent. I will remove this if you wish